app/Contact.php

```diff
@@ -884,6 +884,25 @@ public function getAvatarSource()
         return 'internal';
     }
 
+    /**
+     * Update the gravatar, using the firt email found.
+     */
+    public function updateGravatar()
```

Comment:
We could unit test the new methods in `Contact.php`, what do you think?

Reply:
Yes that's why I've added "TODO" on the comment of the PR ;)

Reply:
I've added some tests